src/app/globals.css

```diff
@@ -0,0 +1,42 @@
+:root {
```

Comment:
Let's keep only single CSS file across the project

Reply:
Done